star_tides/api/controllers/auth_controller.py

```diff
@@ -1,19 +1,25 @@
 ''' star_tides.api.controllers.auth_controller
 '''
+from http import HTTPStatus
 from star_tides.api.controllers.base_controller import Controller
-from star_tides.core.actions.login_user_action import LoginUserAction, CreateUserAction
+from star_tides.core.actions.login_user_action import (
+    LoginUserAction,
+    CreateUserAction
+)
 
 
 class LoginController(Controller):
     ''' Controller to handle basic auth login (username and password)
     '''
-    def process_request(self) -> dict:
+    def process_request(self) -> tuple:
         username, password = self.process_basic_auth()
         jwt, refresh = LoginUserAction(username, password).execute()
-        return {
+        response = {
             'jwt': jwt,
             'refresh_token': refresh
         }
+        http_code = HTTPStatus.OK
+        return response, http_code
```

Comment:
I think the code should be part of the response (default to 200 for OK responses, use the code in the exception for errors).